lib/tasks/gen.tarball.ex

```diff
@@ -0,0 +1,96 @@
+defmodule Mix.Tasks.Supavisor.Gen.Tarball do
```

Comment:
I believe the same can be achieved by adding `:tar` to release `:steps` in mix.exs?

Reply:
Oh, interesting, I'm gonna read mix manual 🤔

Reply:
Yup!

```elixir
  def project do
    [
      # ...
      releases: [
        foo: [
          steps: [
            :assemble,
            &foo/1,
            :tar
          ]
        ]
      ]
    ]
  end

  defp foo(release) do
    Mix.Task.run("eval", ["IO.inspect 1 + 2"])
    release
  end
```